indico/queries/workflow.py

```diff
@@ -266,18 +270,22 @@ def __init__(
         submission: bool = True,
         bundle: bool = False,
         result_version: str = None,
+        streams: Dict[str, io.BufferedIOBase] = None
```

Comment:
Can this keys for this dict be file paths?

Reply:
It could be if that's a requirement, and we could add some logic similar to the way we parse file names out for file paths, but ideally it would just be filename.ext